tests/Query/Mysql/DateDiffTest.php

```diff
@@ -18,9 +19,14 @@ public function testDateDiff()
     public function testDateDiffWithColumnAlias()
     {
         $q = $this->entityManager->createQuery("SELECT DATEADD(p.created, 1, 'DAY') as modified_date FROM DoctrineExtensions\Tests\Entities\Date as p WHERE DATEDIFF(CURRENT_TIME(), modified_date) < 7");
+        if (Version::VERSION >= 2.5) {
+            $alias = 'sclr_0';
+        } else {
+            $alias = 'sclr0';
+        }
```

Comment:
We actually handle this globally with a custom `assertEquals` – basically use the new syntax (with `_`) and it'll get stripped in the tests on older versions.

https://github.com/beberlei/DoctrineExtensions/blob/master/tests/Query/DbTestCase.php#L31-L39